DEVELOPER_GUIDE.md

```diff
@@ -55,7 +55,32 @@ Bound addresses will then be logged to the terminal :
 
 ## Run OpenSearch
 
-- Start a seperate terminal and navigate to the directory that OpenSearch has been cloned to using `cd OpenSearch`
+- Start a separate terminal and navigate to the directory that OpenSearch has been cloned to using `cd OpenSearch`.
+
+- Check if extensions directory exists in OpenSearch using `ls`.
+- If the directory does not exist, create it using `mkdir extensions`.
+- Navigate to the extensions folder using `cd extensions`.
+- Manually create a file titled `extensions.yml` within the extensions directory using an IDE or an in-line text editor.
+
+Sample extensions.yml file:
```

Comment:
I wasn't able to get the extension to load from the default paths, on my machine it was generating this message,
```
[...] Extensions Config Directory :/local/home/petern/git/opensearch/distribution/archives/linux-tar/build/install/opensearch-3.0.0-SNAPSHOT/extensions.
```

I'm not sure if this is expected, but it wasn't picking up the file created off of `{projectRoot}/extensions/extensions.yml`